### ISSUE TITLE
Convert text-app from chrome app to PWA

### ISSUE THREAD

**Issue by BugsNash** [GoogleChromeLabs/text-app#344]
Chrome apps are [deprecated](https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html), so this should be converted to a [PWA](https://developers.google.com/web/progressive-web-apps/).

This is blocked on PWAs implementing a File API.

**Comment by BugsNash** [comment]
Yeah this would definitely have to wait for the file api to be completed. There are no immediate plans to convert text app to pwa.

**Comment by gdluca** [comment]
Any updates for the API? The end of ChromeOS Apps is near and I really would like to continue using this text editor!

Thank you.

**Comment by zainafzal08** [comment]
There is work in progress to move the text app to no longer be a chrome app, this work should be completed before chrome apps end of life :)

**Comment by gdluca** [comment]
Thank you!

**Comment by sanicki** [comment]
Any update on this?

**Comment by nmcain** [comment]
It seems that the File API is somewhat production-ready now

**Comment by rayfoss** [comment]
Latest on this:
- https://googlechromelabs.github.io/text-editor/
- https://github.com/GoogleChromeLabs/text-editor

It doesn't have syntax highlighting or any registered mime types.
